android/autodispose-android/build.gradle

```diff
@@ -33,6 +33,10 @@ android {
     sourceCompatibility JavaVersion.VERSION_1_7
     targetCompatibility JavaVersion.VERSION_1_7
   }
+  // TODO replace with https://issuetracker.google.com/issues/72050365 once released.
+  libraryVariants.all {
```

Comment:
can you do it at top level build.gradle instead of duplicating please?

Reply:
Good point, done in 5fc74b8

Note the `afterEvaluate` is necessary